dev-duckies-sm-proj/src/pages/register-page/register.css

```diff
@@ -0,0 +1,14 @@
+.card {
+    display: flex;
+    width: 400px;
+    height: 500px;
+    margin: auto;
+    margin-top: 200px;
+}
+
+.card-content{
+    display: flex;

```

Comment:
Correct me if I'm wrong, but since the class 'card-content' is nested inside the 'card' class, wouldn't the second flex property be redundant? Not trying to be picky, this is more of me thinking out loud. Great job though!